kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue

```diff
@@ -159,7 +169,7 @@
         useDevicesResult = useDevicesWithChannel(props.filterByChannelId);
       } else if (props.filterByFacilityId !== null) {
         // This is inherently filtered to full-facility devices
-        useDevicesResult = useDevicesWithFacility(props.filterByFacilityId);
+        useDevicesResult = useDevicesWithFacility({ facilityId: props.filterByFacilityId });
```

Comment:
I guess this also opens the way for filtering by devices with facilities with specific characteristics/facility settings too?